### PR TITLE
feat Added Hamburger component

### PR DESCRIPTION
**Pull request by ayush65**

### CODE REVIEW

**Review by shubhambishnoi — APPROVED** [review]

**Review by JowelTisso — APPROVED** [review]

**Review by rahulmistry751 — APPROVED** [review]

**Review by Himanshu-Dhawale — APPROVED** [review]

File: components/Badge/Badge.css [hunk 1]
```diff
@@ -7,7 +7,7 @@
     height: 93.75rem;
 }
 .component-library-components-container{
-    margin: 6.25rem auto 2.25rem 5.875rem; /* margin-top  margin-right margin-bottom margin-left  */
+    margin: 6.25rem auto 2.25rem 21.875rem; /* margin-top  margin-right margin-bottom margin-left  */
```
Comment (anjali1102) [inline components/Badge/Badge.css]: can you remove the commented css

**Review by anjali1102 — APPROVED** [review]
LGTM !
no as such suggestions

File: components/Badge/Badge.css (hunk 1 above)
Comment (ayush65) [reply components/Badge/Badge.css]: yeaa i have done it thanks for reviewing

**Review by anjali1102 — APPROVED** [review]
👍

**Pull request merged**